ui/pages/multi-srp/import-srp/import-srp.tsx

```diff
@@ -230,132 +233,164 @@ export const ImportSrp = ({
-            actionButtonLabel={t('clear')}
-            actionButtonOnClick={() => {
-              onSrpChange(Array(defaultNumberOfWords).fill(''));
-              setSrpError('');
-            }}
-            data-testid="bannerAlert"
-          />
-        ) : null}
-
-        {
-          <Box width={BlockSize.Full} marginTop={4}>
-            <ButtonLink
-              width={BlockSize.Full}
-              loading={loading}
-              onClick={async () => {
-                setNumberOfWords(numberOfWords === 12 ? 24 : 12);
+              );
+            })}
+          </Box>
+          {srpError ? (
+            <BannerAlert
+              severity={BannerAlertSeverity.Danger}
+              description={srpError}
+              actionButtonLabel={t('clear')}
+              actionButtonOnClick={() => {
+                onSrpChange(Array(defaultNumberOfWords).fill(''));
                 setSrpError('');
-                setInvalidSrpWords(
-                  Array(numberOfWords === 12 ? 24 : 12).fill(false),
-                );
               }}
-              data-testid="import-srp__multi-srp__switch-word-count-button"
-            >
-              {t('importNWordSRP', [numberOfWords === 12 ? '24' : '12'])}
-            </ButtonLink>
-          </Box>
-        }
-      </Box>
-      <Box
-        className="import-srp__multi-srp__import-button"
-        width={BlockSize.Full}
-        marginTop={4}
-        paddingBottom={6}
-        paddingTop={2}
-        backgroundColor={
-          theme === ThemeType.light
-            ? BackgroundColor.backgroundDefault
-            : BackgroundColor.backgroundDefault
-        }
-      >
-        <ButtonPrimary
-          width={BlockSize.Full}
-          disabled={!isValidSrp || hasEmptyWordsOrIncorrectLength}
-          loading={loading}
-          onClick={async () => {
-            try {
-              setLoading(true);
-              await importWallet();
-              onActionComplete(true);
-              dispatch(setShowNewSrpAddedToast(true));
-            } catch (e) {
-              setSrpError(
-                e instanceof Error ? e.message : 'An unknown error occurred',
-              );
-              setLoading(false);
-            }
-          }}
+              data-testid="bannerAlert"
+            />
+          ) : null}
+
+          {
+            <Box width={BlockSize.Full} marginTop={4}>
+              <ButtonLink
+                width={BlockSize.Full}
+                loading={loading}
+                onClick={async () => {
+                  setNumberOfWords(numberOfWords === 12 ? 24 : 12);
+                  setSrpError('');
+                  setInvalidSrpWords(
+                    Array(numberOfWords === 12 ? 24 : 12).fill(false),
+                  );
+                }}
+                data-testid="import-srp__multi-srp__switch-word-count-button"
+              >
+                {t('importNWordSRP', [numberOfWords === 12 ? '24' : '12'])}
```

Comment:
Nit: This logic was already like this before, but I think he'd be great to avoid repetition and add a `const otherNumberOfWords = numberOfWords === 12 ? 24 : 12;` (or another variable name 😄) above, so we could use like:

```suggestion
                  setNumberOfWords(otherNumberOfWords);
                  setSrpError('');
                  setInvalidSrpWords(
                    Array(otherNumberOfWords).fill(false),
                  );
                }}
                data-testid="import-srp__multi-srp__switch-word-count-button"
              >
                {t('importNWordSRP', [`${otherNumberOfWords}`])}
```